src/main/twirl/uk/gov/hmrc/govukfrontend/views/layouts/govukTemplate.scala.html

```diff
@@ -66,7 +66,10 @@
     @skipLinkBlock.getOrElse {
       @govukSkipLink(SkipLink(
         href = "#main-content",
-        content = HtmlContent("Skip to main content")
+        content = HtmlContent(htmlLang match {
```

Comment:
Any reason why this is HtmlContent not TextContent? (I know that was already there but it looks like it might have been the wrong choice.  TextContent should be encoded on its way to the page where HtmlContent won't be.

Reply:
✅  refactored to Text